### PR TITLE
adds Pavian

### LINKED ISSUE #764: [Container Request]: Pavian
### What tool are you looking for and why?

Pavian is a interactive browser application for analyzing and visualization metagenomics classification results from classifiers such as Kraken, KrakenUniq, Kraken 2, Centrifuge and MetaPhlAn. Pavian also provides an alignment viewer for validation of matches to a particular genome.

publication: https://doi.org/10.1093/bioinformatics/btz715

GitHub: https://github.com/fbreitwieser/pavian

### PR DESCRIPTION
**Pull request by Kincekara**
Closes #764

Pull Request (PR) checklist:
- [x] Include a description of what is in this pull request in this message.
- [x] The dockerfile successfully builds to a test target for the user creating the PR. (i.e. `docker build --tag samtools:1.15test --target test docker-builds/samtools/1.15` )
- [x] Directory structure as name of the tool in lower case with special characters removed with a subdirectory of the version number (i.e. `spades/3.12.0/Dockerfile`)
   - [ ] (optional) All test files are located in same directory as the Dockerfile (i.e. `shigatyper/2.0.1/test.sh`)
- [x] Create a simple container-specific [README.md](https://github.com/StaPH-B/docker-builds/blob/master/.github/workflow-templates/readme-template.md) in the same directory as the Dockerfile (i.e. `spades/3.12.0/README.md`)
   - [ ] If this README is longer than 30 lines, there is an explanation as to why more detail was needed
- [x] Dockerfile includes the recommended [LABELS](https://github.com/StaPH-B/docker-builds/blob/master/dockerfile-template/Dockerfile#L8-L18) 
- [x] Main [README.md](https://github.com/StaPH-B/docker-builds/blob/master/README.md) has been updated to include the tool and/or version of the dockerfile(s) in this PR
- [x] [Program_Licenses.md](https://github.com/StaPH-B/docker-builds/blob/master/Program_Licenses.md) contains the tool(s) used in this PR and has been updated for any missing

Notes:
**Base:** I used rocker/shiny:4.1.0, based on Ubuntu focal. Since the app uses an older version of Rsamtools, jammy, and jammy-based rocker images didn't work. 
**Release:** The latest version 1.2.1 is not released. There are a lot of improvements since version 1, so I used the latest commit instead of the release
**Test:** Since the app works on a web browser, I couldn't think of a good test. I listed the installed R packages in the test stage. Please let me know if you have a better idea.   
**Bonus:** The image works on gitpod, too. You can use my test image kincekara/pavian:1.2.1

### CODE REVIEW

File: pavian/1.2.1/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,40 @@
+FROM rocker/shiny:4.1.0 as app
```
Comment (erinyoung) [inline pavian/1.2.1/Dockerfile:1]: Nice! An Rshiny app!

File: pavian/1.2.1/README.md [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+Code repository: https://github.com/fbreitwieser/pavian
+
+Additional tools:
+- R: 4.1.0
+
+Basic information on how to use this tool:
+
+Pavian is a interactive browser application for analyzing and visualization metagenomics classification results from classifiers such as Kraken, KrakenUniq, Kraken 2, Centrifuge and MetaPhlAn. It also provides an alignment viewer for validation of matches to a particular genome.  
+
+![image](https://cloud.githubusercontent.com/assets/516060/20188595/5c8b9808-a747-11e6-9235-296a2314659a.png)
```
Comment (erinyoung) [inline pavian/1.2.1/README.md:15]: I like the image in the readme

**Review by erinyoung — APPROVED** [review]
Looked good in a live demo!

I hope not too many people get IT security warnings.

**Comment by erinyoung** [comment]
You can check the status of the deploy here : https://github.com/StaPH-B/docker-builds/actions/runs/6749644898

**Comment by Kincekara** [comment]
Thank you!

**Pull request merged**